app.py

```diff
@@ -1,80 +1,101 @@
-'''
+"""
 Flask Application
-'''
+"""
+
 from flask import Flask, jsonify, request
 from models import Experience, Education, Skill
 
 app = Flask(__name__)
 
 data = {
     "experience": [
-        Experience("Software Developer",
-                   "A Cool Company",
-                   "October 2022",
-                   "Present",
-                   "Writing Python Code",
-                   "example-logo.png")
+        Experience(
+            "Software Developer",
+            "A Cool Company",
+            "October 2022",
+            "Present",
+            "Writing Python Code",
+            "example-logo.png",
+        )
     ],
     "education": [
-        Education("Computer Science",
-                  "University of Tech",
-                  "September 2019",
-                  "July 2022",
-                  "80%",
-                  "example-logo.png")
+        Education(
+            "Computer Science",
+            "University of Tech",
+            "September 2019",
+            "July 2022",
+            "80%",
+            "example-logo.png",
+        )
     ],
-    "skill": [
-        Skill("Python",
-              "1-2 Years",
-              "example-logo.png")
-    ]
+    "skill": [Skill("Python", "1-2 Years", "example-logo.png")],
 }
 
 
-@app.route('/test')
+@app.route("/test")
 def hello_world():
-    '''
+    """
     Returns a JSON test message
-    '''
+    """
     return jsonify({"message": "Hello, World!"})
 
 
-@app.route('/resume/experience', methods=['GET', 'POST'])
+@app.route("/resume/experience", methods=["GET", "POST"])
 def experience():
-    '''
+    """
     Handle experience requests
-    '''
-    if request.method == 'GET':
+    """
+    if request.method == "GET":
         return jsonify()
 
-    if request.method == 'POST':
-        return jsonify({})
+    if request.method == "POST":
```

Comment:
Is there a separate issue for adding test for this endpoint? If not, let's add a test in this PR